portal/jam-embedded-api.md

```diff
@@ -278,45 +240,253 @@ The snippet uses a sample `index.html` page. Note that you will have to replace
+    document.getElementById('jam-share-url').innerHTML = 'Please wait! Uploading the log and generating share URL ...';
+
+    // The jam.share() method accepts argument of type { workspaceId: string, submittedBy: string, password: string }
+    const jamShareUrl = await jam.share(); // when the ShareOptions are omitted, the is uploaded to the default organization workspace without encryption protection.
+    document.getElementById('capture-info').innerHTML = 'Log generation completed! (jam.state = ' + jam.state + ')';
+    document.getElementById('jam-share-url').innerHTML = 'Share URL: ' +  '<a href="' + jamShareUrl + '" target="_blank">'+ jamShareUrl +'</a>';
+    document.getElementById('last-error').innerHTML = 'Use "Reset Capture" to clean and prepare Jam Embedded for new capture';
+    document.getElementById('last-error-info').innerHTML = '';
+
+    await navigator.clipboard.writeText(jamShareUrl);
+}
+
+async function reset() {
+    document.getElementById('btn-start').disabled = false;
+    document.getElementById('btn-start-video').hidden = true;
+    document.getElementById('btn-stop').disabled = true;
+    document.getElementById('btn-share').disabled = true;
+
+    await jam.reset();
+
+    document.getElementById('capture-info').innerHTML = 'Fiddler Jam Embedded reset and ready for new capture! (jam.state = ' + jam.state + ')';
+    document.getElementById('jam-share-url').innerHTML = '';
+    document.getElementById('last-error').innerHTML = '';
+    document.getElementById('last-error-info').innerHTML = '';
+}
 ```
 
-## Limitations and Browser Specifics
+The above snippets create a basic HTML page that utilizes most Fiddler Jam Embedded functionalities. Different browsers, like Edge, Chrome, Brave, and other non-Chromium browsers such as Firefox and Safari, may show behavioral differences.
+
+## Generating API Key
+
+You need a unique API key to use the Fiddler Jam Embedded API. The API key will allow you to initialize the Jam Embedded script, upload recorded logs to your organizational workspaces, and generate public or protected share links. Only licensed portal users can request Fiddler Jam quota and thus render their unique organizational API key. Follow these steps to generate your Fiddler Jame Embedded API key:
+
+- Contact us and request your Fiddler Jam Embedded quota. Proceed to the next step only once your organizational quota is set.
+
+- Login into the [Fiddler Jam portal](https://jam.getfiddler.com).
+
+- Navigate to the Fiddler Jam Portal menu (top-right corner) and follow the **Settings > Embedded** (the option is available only after the organizational quota is set).
+
+- Click on **API Key: Generate** link.
+
+- Copy your Fiddler Jam Embedded API key.
+
+If you want to regenerate the key, please get in touch with us at [support@getfiddler.com](mailto:support@getfiddler.com).
+
+## Limitations and Log Specifics
```

Comment:
Can you please add information that masking is not supported for screenshots and also that there's a performance issue when capturing screenshots for large applications.